README.md

```diff
@@ -97,40 +105,59 @@ public class MainApplication extends Application implements ReactApplication {
 
 ## Example
 ```javascript
-var Mailer = require('react-native-mail');
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import { View, Alert, Button } from 'react-native';
+import Mailer from 'react-native-mail';
 
-var MailExampleApp = React.createClass({
-  handleHelp: function() {
+export default class App extends Component {
+
+  handleEmail = () => {
     Mailer.mail({
       subject: 'need help',
       recipients: ['support@example.com'],
       ccRecipients: ['supportCC@example.com'],
       bccRecipients: ['supportBCC@example.com'],
-      body: '',
+      body: '<b>A Bold Body</b>',
       isHTML: true,
       attachment: {
         path: '',  // The absolute path of the file from which to read data.
         type: '',   // Mime Type: jpg, png, doc, ppt, html, pdf
         name: '',   // Optional: Custom filename for attachment
       }
     }, (error, event) => {
-        if(error) {
-          AlertIOS.alert('Error', 'Could not send mail. Please send a mail to support@example.com');
-        }
+      Alert.alert(
+        'Error',
+        'Email could not be sent',
```

Comment:
@geirman sorry for bikeshedding this. wondering if we should simply show an alert with `error` and `event` in the message. Title as `Error` shows negative ideas. thoughts?